packages/derive/src/lib.rs

```diff
@@ -136,6 +198,40 @@ mod test {
 
     use crate::entry_point_impl;
 
+    #[test]
+    fn contract_state_version_expansion() {
+        let code = quote! {
+            #[set_contract_state_version(version = 2)]
```

Comment:
Can we get rid of one of the `version`s? Seems somewhat redundant

Reply:
Sure thing. Got rid of `version =`. It now takes the integer literal directly.